### DISCUSSION TITLE
OR between a specific martensite variant and and particular parent Grain

### DISCUSSION THREAD

**Discussion by Sofianehmz** [mtex-toolbox/mtex#1855] [Ask Anything]
Hello,

I need to calculate the OR relationship between a specific martensite variant and a corresponding parent austenite grain. I have the IDs of both, the grains (childs) and the ParentGrains but the 'job' function dosen't seem to work on individual grains(If assigned with IDs). Should I determine  the crystal symmetry of of the variant and the parent grain then calculate the OR ? 
Thanks in advance

**Comment by AzdiarGazder** [comment]
Hi @Sofianehmz,

To help reconstruct your parent microstructure, or work with single parent grains, please refer to  the following weblink: https://github.com/ORTools4MTEX/ORTools.

Warm regards,
Azdi
